.changes/unreleased/security-20240812-133345.yaml

```diff
@@ -0,0 +1,3 @@
+kind: security
+body: removed high level snyk issues for hard coded credentials
```

Comment:
Please fix so it's a sentence with punctuation. Will be a bullet point in changelog

Reply:
done :)